### PR TITLE
Add SMS OTP Configuration UI and the Ability To Setup SMS OTP via Application Sign-In Method Landing Page

### PR DESCRIPTION
**Pull request by nipunsampath**
- A card for SMS OTP at the **Connections** screen to configure SMS OTP. 
- A UI for configuring SMS OTP. 
- A card at an application's Sign-In method landing page to setup SMS OTP in the application.

### Related Issues
- https://github.com/wso2-enterprise/iam-engineering/issues/160

### Related PRs
- https://github.com/wso2-enterprise/asgardeo-app-extensions/pull/1941

### CODE REVIEW

File: apps/console/src/extensions/configs/authenticator.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 Inc. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ * You may not alter or remove any copyright or other notice from copies of this content."
+ */
```
Comment (thivi) [inline apps/console/src/extensions/configs/authenticator.ts:8]: Shall we change the license header? Check the other files as well.

File: apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx [hunk 2]
```diff
@@ -120,117 +120,64 @@ export const SignInMethodLanding: FunctionComponent<SignInMethodLandingPropsInte
...
-                                ) }
+                                )}
                             </Heading>
-                            { !hiddenOptions.includes(LoginFlowTypes.SECOND_FACTOR_TOTP) && (
+                            {!hiddenOptions.includes(LoginFlowTypes.SECOND_FACTOR_TOTP) && (
                                 <InfoCard
                                     fluid
                                     data-testid="totp-mfa-flow-card"
-                                    image={ getAuthenticatorIcons().totp }
+                                    image={getAuthenticatorIcons().totp}
```
Comment (thivi) [inline apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx:135]: Let's make sure there is a space after and before `{` and `}`.

**Review by thivi — APPROVED** [review]

**Review by thivi — APPROVED** [review]

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx [hunk 3]
```diff
@@ -306,6 +307,14 @@ export const SignOnMethods: FunctionComponent<SignOnMethodsPropsInterface> = (
                 ...authenticationSequence,
                 ...cloneDeep(SecondFactorEMAILOTPSequenceTemplate)
             });
+        } else if(loginFlow === LoginFlowTypes.SECOND_FACTOR_SMS_OTP){
```
Comment (LakshiAthapaththu) [inline apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx:310]: > else if(loginFlow === LoginFlowTypes.SECOND_FACTOR_SMS_OTP) {

Comment (LakshiAthapaththu) [inline apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx:310]: Check formatting

Comment (nipunsampath) [reply apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx:310]: Fixed with 57492cfc2bc379354647133894224a4278beddb0

**Pull request merged**